app/models/user_access.rb

```diff
@@ -97,6 +97,24 @@ def accessible_users(action)
       .where(phone_number_authentications: {registration_facility_id: accessible_facilities(action)})
   end
 
+  def accessible_protocols(action)
+    return Protocol.none unless [:manage].include?(action)
```

Comment:
Why return none if you ask for protocols you can view? Can we simplify this to:

```
return Protocol.all if power_user?
return Protocol.all if accessible_organizations(:manage).any?

Protocol.none
```

Reply:
Hmm. Can do this, however, there isn't a simple `view` action. The other supported actions (`view_reports`, `view_pii`, and `manage_overdue_list`) don't really make sense in the context of protocols or protocol drugs.

We could add a `view` action, but we don't currently have a case for `view`ing protocols or drugs you can't manage.

Reply:
I'm sorry, I shouldn't have use the word "view". I understand that protocol-related models are "all or nothing". Either you can manage all of them, or you can't do anything. My question was more scoped to the logic of this method. Can we simplify? ☝️

Reply:
Speaking of viewing protocols, one touchpoint for viewing protocols is selecting one for your facility group, right? Are we planning to expose `ProtocolDrug.all` in the selection dropdown? I think that's ok, as long as we're making that conscious decision.

Reply:
Ah, I misread the question. wdyt of:
```
return Protocol.all if power_user?
return Protocol.all if 𝗮𝗰𝘁𝗶𝗼𝗻 == :𝗺𝗮𝗻𝗮𝗴𝗲 && accessible_organizations(:manage).any? 
Protocol.none
```
 Mostly so we prevent usages like `accessible_protocols(:view_pii)` which don't make sense.

Reply:
> Speaking of viewing protocols, one touchpoint for viewing protocols is selecting one for your facility group, right? Are we planning to expose ProtocolDrug.all in the selection dropdown? I think that's ok, as long as we're making that conscious decision.

Yup, this is the existing behaviour

Reply:
Yup that looks perfect 👍